docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,247 @@
+   
+   - To save a card during the transaction, sign in to the BigCommerce store as a shopper, or create a new account during checkout. In the payments step, select **Save this card for later**. 
+
+## Declare environment variables
+
+<!-- theme: info -->
+> #### Note on naming convention
+> In some places, this guide and app template code refer to a Stripe API account's public key as a client ID, and its secret key as a client secret.
+
+1. Create a `.env` file in the root directory of your project.
+
+2. Copy the content of `.env-sample` to `.env`.
+
+```shell title="Copy .env-sample contents"
+cp .env-sample .env
+```
+Update the following environment variables:
+
+| Environment variable | Description | Reference location |
+|:---------------------|:------------|:-------------------|
+| `NEXT_PUBLIC_APP_URL` | A public-facing URL that can receive webhooks. | On this page, see [Create an HTTPS tunnel](#create-an-https-tunnel); see also [the HTTPS tunnel section of the Sample App Tutorial](/api-docs/apps/tutorials/sample-app-nextjs/step-2-connect#create-an-https-tunnel?source=subscription-foundation)  |
+| `NEXT_PUBLIC_APP_ID` | The app's ID | [Find an App's ID](/api-docs/apps/tutorials/id#find-in-developer-portal?source=subscription-foundation) |
+| `BC_APP_CLIENT_ID` | The app API account's client ID | [View App Credentials](/api-docs/apps/guide/developer-portal#view-credentials?source=subscription-foundation) |
+| `BC_APP_SECRET` | The app API account's client secret | [View App Credentials](/api-docs/apps/guide/developer-portal#view-credentials?source=subscription-foundation) |
+| `NEXT_PUBLIC_STRIPE_CLIENT_ID` | The app-specific Stripe Connect API account client ID | see [Enable Stripe Connect for Platforms](#enable-stripe-connect-for-platforms) |
+| `STRIPE_SECRET_KEY` | The app-specific Stripe Connect API account client secret | see [Create Stripe Accounts](#create-stripe-accounts) |
+
+
+## Run migration and start the server
+
+<!-- theme: info -->
+> #### Database note
+> This example uses SQLite as a data store. In production, we recommend using a database that has more robust concurrency support, such as PostgreSQL. For information on switching databases, see the [Replacing SQLite](#replacing-sqlite) section.
+
+1. If you're using SQLite, which is Subscription Foundation's default data store, skip to the next step. Otherwise, see [Replacing SQLite](#replacing-sqlite)
+
+2. Run the pre-configured Prisma migration to create the database tables and initial client as defined in `/prisma/migrations/*`. Troubleshoot as needed.
+
+```shell title="Run Prisma migration"
+npx prisma migrate dev
+```
+
+![npx_prisma_migrate](https://storage.googleapis.com/bigcommerce-production-dev-center/images/npx_prisma_migrate.png)
+
+3. Start the app server.
+
+```shell title="Start the app server"
+npm run dev
+```
+
+4. Start ngrok using the follow command. If your app does not run on port 3000, replace `3000` with your app server's port.
```

Comment:
We tell readers to start ngrok twice, here and on https://github.com/bigcommerce/dev-docs/pull/1551/files#diff-afbc8e60ab383e37072f5455af61ac5e9c3c87f0138896c79a7861d5a4af68ecR37. 

The problem with that is that they need to start ngrok at the beginning to get the ID for the `https://{ngrok_id}.ngrok.io` address, update the `.env` file, the BC app profile, Stripe, and whichever other places. If they run it at the end, ngrok will assign a new ID and they'll need to update in all those places.